.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
+  "printWidth": 120,
+  "tabWidth": 2,
+  "semi": false,
+  "singleQuote": true,
+  "trailingComma": "all",
```

Comment:
Like this, in the [docs](https://prettier.io/docs/en/options.html#trailing-commas) it is stated that this needs Node 8+, just for clarification, this would nevertheless not prevent us to test (and locally debug) code with Node 6, would it?

Reply:
You always transpile typings to js. `ts-node` does it on the fly. So you run whatever is specified in tsconfig's target.